platform/linuxbsd/crash_handler_linuxbsd.cpp

```diff
@@ -61,12 +63,15 @@ static void handle_crash(int sig) {
 	}
 
 	// Dump the backtrace to stderr with a message to the user
+	fprintf(stderr, "\n================================================================\n");
 	fprintf(stderr, "%s: Program crashed with signal %d\n", __FUNCTION__, sig);
 
 	if (OS::get_singleton()->get_main_loop()) {
 		OS::get_singleton()->get_main_loop()->notification(MainLoop::NOTIFICATION_CRASH);
 	}
 
+	// Print the engine version just before, so that people are reminded to include the version in backtrace reports.
+	fprintf(stderr, "Engine version: " VERSION_FULL_NAME " (" VERSION_HASH ")\n");
```

Comment:
I think `VERSION_HASH` stays empty on official tagged releases, so this might need to be conditional.